### DISCUSSION TITLE
Images not rendered in Pagedjs PDF

### DISCUSSION THREAD

**Discussion by 1000camels** [thegetty/quire#912] [Get Help]
My images in my PDF are not rendered, with the exception of images set in css using the background-image property.
The only difference I can see is that I am using a sub-directory in my path, because I am publishing to github.io. I have set the url in config.yaml.

Any suggestions on how to resolve this?

**Comment by 1000camels** [comment]
I have discovered that the issue is my url setting in publications.yaml. I have it set to the final url I am using on github.io, not 'http://localhost:8080'.
As a test, I switched it back to the default, and re-ran quire build and quire pdf and my images appear. I can obviously do this in my workflow, but it seems like when pdf.html is produced, it should always use 'http://localhost:8080'

**Comment by Erin-Cecele** [comment]
Hi @1000camels, can you please run `quire info --debug` in your project and share the output here? Thank you!

**Comment by 1000camels** [comment]
[biophilia]
 quire-cli 1.0.0-rc.10
 quire-11ty 1.0.0-rc.16
 starter https://github.com/thegetty/quire-starter-default@2.6.0
[System]
 quire-cli 1.0.0-rc.11
 node v18.17.1
 npm 10.2.4
 os Darwin 22.6.0

**Comment by 1000camels** [comment]
I think you can reproduce this error by changing the publications url setting to something with a subdomain. Even this would show the broken images:

http://localhost:8080/project-name/'

**Comment by Erin-Cecele** [comment]
Thank you @1000camels. This may be related to known/existing issue. I will test and confirm.

**Comment by Erin-Cecele** [comment]
Hi @1000camels, thank you for sharing this with us. As I understand it, this is a known issue that will be fixed in our next release. I can ping you when that release happens. Thank you!
